### ISSUE TITLE
reset splashpage button or command

### ISSUE THREAD

**Issue by SilvanPixel** [mazi-project/portal#148]
Once the splashpage has been visited by a device it cant be shown again. Would be great to have a reset button in the toolkit or an command line to reset the splashpage so it shows again on all devices.

**Comment by panosnethood** [comment]
Yes, many people have had this request recently. I guess it would be nice to be able to both manually reset the splash page for all devices and configure the time threshold that this happens automatically.

Related to this, but on the other extreme, in my own permanent installation it happens often that I get disconnected from the Internet because my "registration" has expired and it is not made obvious why this is the case. I think it would be nice when this happens users to be redirected again to the splash page instead of just getting disconnected.

@mavridis92 what do you think? Is this easy to implement or at least give instructions on how to administer important parameters of nodogsplash through the command line?

**Comment by mavridis92** [comment]
Bellow, you will find the reset procedure of the nodogsplash service. 
1) Connected to the Toolkit.
ssh pi@10.0.0.1 
2) Get root privileges 
sudo su
3)Delete the Authenticated users file. 
rm /etc/mazi/users.dat
4)Restart the nodogsplash service.
/etc/init.d/nodogsplash stop
/etc/init.d/nodogsplash start

  Regarding the reappearance of the splash page at the end of registration, one simple solution could be that to disconnect the user as you mention, but before we decide, I recommend investigating the opportunities of the nodogsplash v2 which is going to be installed to the new image Stretch.

**Comment by haniavis** [comment]
Hi,

what's the reason behind reseting the splash page? For debugging purposes I think it is better to do it through command-line, otherwise which is the case you need it?

**Comment by mgaved** [comment]
This is useful for me. I am manually editing the splash page with some custom text and formatting so (@haniavis ) I want to be able review what I have done and play with it until the new splash screen looks how my community group wants it to look. At the moment this is fine to do using the command line but for a more accessible splash screen that can be edited by a wider number of people I would like the team to enable editing via easier tools. Thanks!

**Comment by panosnethood** [comment]
Did you try this URL? http://local.mazizone.eu:4567/splash

**Comment by mgaved** [comment]
Thanks Panosnethood - to clarify, I think you're saying going to http://local.mazizone.eu:4567/splash allows us to see the splashscreen again on the same device even though the device has logged in and the lease period has not expired (so this enables checking of edits to the html, etc.  of the splash page)
